src/app/content/hooks/locationChange.ts

```diff
@@ -1,22 +1,33 @@
+import css from 'cnx-recipes/styles/output/intro-business.json';
 import { routeHook } from '../../navigation/utils';
 import { receiveBook, receivePage, requestBook, requestPage } from '../actions';
 import { content } from '../routes';
 import * as select from '../selectors';
 import { archiveLoader } from '../utils';
 
-export default routeHook(content, ({dispatch, getState}) => async({match}) => {
+const fontMatches = css.match(/"(https:\/\/fonts\.googleapis\.com\/css\?family=.*?)"/);
+const fonts = fontMatches ? fontMatches.slice(1) : [];
+
+export default routeHook(content, ({dispatch, getState, fontCollector}) => async({match}) => {
   const state = getState();
   const {bookId, pageId} = match.params;
   const book = select.book(state);
   const page = select.page(state);
+  const promises: Array<Promise<any>> = [];
+
+  fonts.forEach((font) => fontCollector.add(font));
```

Comment:
invented font collector. dual purpose:
- pre-render script will hoist these fonts into the head so we aren't relying on styled-components `@include` polyfill to fetch them from google.
- can hook into when the fonts are loaded